### ISSUE TITLE
Daemon is not restarted if stops

### ISSUE THREAD

**Issue by jeremypw** [elementary/calendar#368]
Not sure if this is an issue, but I noticed that if the daemon is stopped for any reason, it is not automatically restarted even if new events are added with the main app.  So reminders etc will not appear.

**Comment by danrabbit** [comment]
Yeah we should definitely make sure the daemon is always running for reminders

**Comment by jeremypw** [comment]
So, add it to the cerbere watch list?

**Comment by danrabbit** [comment]
@jeremypw I think there's a more modern way to do that with systemd or something isn't there?

**Comment by jeremypw** [comment]
Quite possibly, but cerbere is already being used so is the quickest, easiest fix.

**Comment by danrabbit** [comment]
@codygarver might know more, but I believe the plan was to get away from using Cerbere. I think, for example, stuff like the agents aren't using Cerbere

**Comment by jeremypw** [comment]
Ah, OK - I'll investigate what other mechanism is being used elsewhere.
